modules/geo-layers/src/tileset-2d/tile-2d-header.ts

```diff
@@ -78,7 +78,9 @@ export class Tile2DHeader<DataT = any> {
   }
 
   get isLoaded(): boolean {
-    return this._isLoaded && !this._needsReload;
+    return (
+      this._isLoaded && !this._needsReload && Boolean(this.layers?.every(layer => layer.isLoaded))
```

Comment:
This assumes a tile isn't loaded until it has rendered layers.

If a tile can be considered loaded without rendering any layers, the logic could look something like

```
if (!this._isLoaded) return false;
if (this._needsReload) return false;
if (tile.layers) return tile.layers.every(layer => layer.isLoaded);
return true;
```